### PR TITLE
added environment variables

### PR DESCRIPTION
**Pull request by shubhang-eGov**

### CODE REVIEW

**Review by rohit-eGov — APPROVED** [review]

File: deploy-as-code/helm/charts/core-services/egov-url-shortening/values.yaml [hunk 1]
```diff
@@ -62,6 +62,10 @@ env: |
     value: "false"  
   - name: MANAGEMENT_SECURITY_ENABLED
     value: "false"
+  - name: HASHIDS_SALT
+    value: {{ index .Values "egov-url-shorten-hashid-salt" | quote}}
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/core-services/egov-url-shortening/values.yaml:66]: No need for "egov-url-shorten-" prefix.

File: deploy-as-code/helm/charts/core-services/egov-url-shortening/values.yaml [hunk 2]
```diff
@@ -62,6 +62,10 @@ env: |
...
+  - name: HASHIDS_MIN_LENGTH
+    value: {{ index .Values "egov-url-shorten-hashid-min-length" | quote}}
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/core-services/egov-url-shortening/values.yaml:68]: To maintain uniformity, append _S_ to the variable name _hashid_. The the variable name would become **hashids-min-length**

File: deploy-as-code/helm/environments/dev.yaml [hunk 3]
```diff
@@ -127,6 +127,8 @@ egov-filestore:
 
 egov-url-shortening:
   egov-url-shortner-endpoint: "/egov-url-shortening/shortener"
+  egov-url-shorten-hashid-salt: "randomsalt"
+  egov-url-shorten-hashid-min-length: "5"
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/environments/dev.yaml:131]: Does dev cluster need 5? Or would 4 work.

File: deploy-as-code/helm/charts/core-services/egov-url-shortening/values.yaml [hunk 4]
```diff
@@ -62,6 +62,10 @@ env: |
     value: "false"  
   - name: MANAGEMENT_SECURITY_ENABLED
     value: "false"
+  - name: HASHIDS_SALT
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/core-services/egov-url-shortening/values.yaml:65]: Surround these two env vars with if condition.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

**Review by rushang7-eGov — APPROVED** [review]

**Review by sathishp-eGov — APPROVED** [review]

**Pull request merged**